var/spack/repos/builtin/packages/rust/package.py

```diff
@@ -324,6 +332,37 @@ class Rust(Package):
         ]
     }
 
+    # Specifies the strings which represent a pre-release Rust version. These
+    # always bootstrap with the latest beta release.
+    #
+    # NOTE: These are moving targets, and therefore have no stable checksum. Be
+    # sure to specify "-n" or "--no-checksum" when installing these versions.
+    rust_prerelease_versions = ["beta", "nightly", "master"]
+
+    for prerelease_version in rust_prerelease_versions:
+        for rust_target, rust_arch_list in iteritems(rust_archs):
+            for rust_arch in rust_arch_list:
+                # All pre-release builds are built with the latest beta
+                # compiler.
+                resource(
+                    name='rust-beta-{target}'.format(
+                        target=rust_target
+                    ),
+                    url='https://static.rust-lang.org/dist/rust-beta-{target}.tar.gz'.format(
+                        target=rust_target
+                    ),
+                    # Fake SHA - checksums should never be checked for
+                    # pre-release builds, anyway
+                    sha256='0000000000000000000000000000000000000000000000000000000000000000',
```

Comment:
I hate this - similar to above, any way to specify there will never be a checksum?